python/plugins/processing/algs/gdal/helpers/validate_cloud_optimized_geotiff.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python3
```

Comment:
Do we really need to copy this into QGIS source? If this script is part of GDAL distribution better to keep it outside of QGIS tree to reduce maintenance overhead.

Reply:
As far as I understood, it's not part of the distribution, but only a sample file ? At least the file doens't seem present when installed with OSGeo4W.

Reply:
@rouault can you shed some light here?

Reply:
yes, the script is not installed by GDAL. It just lives in its git repository. In principle, the output of the GDAL COG driver should be conformant :-) (the script is used in GDAL autotest suite to check that)